### PR TITLE
[3.6]  bpo-28231: The zipfile module now accepts path-like objects for external paths.

### PR DESCRIPTION
**Pull request by serhiy-storchaka**

### CODE REVIEW

File: Doc/library/zipfile.rst [hunk 1]
```diff
@@ -183,6 +184,9 @@ ZipFile Objects
       Previously, a plain :exc:`RuntimeError` was raised for unrecognized
       compression values.
 
+   .. versionchanged:: 3.6.2
```
Comment (lelit) [inline Doc/library/zipfile.rst:187]: Given that this landed in the 3.6 branch, doesn't that mean that the upcoming 3.6.1 will contain this?

Comment (serhiy-storchaka) [reply Doc/library/zipfile.rst:187]: I think this is too late for 3.6.1. Unless @ned-deily cherry-pick it.

Comment (ned-deily) [reply Doc/library/zipfile.rst:187]: Yes, this is too late for 3.6.1.

Comment (lelit) [reply Doc/library/zipfile.rst:187]: Thank you, it was just my misunderstanding on how the upcoming release will be cut, I assumed it's feed by the `3.6` branch.

Comment (ned-deily) [reply Doc/library/zipfile.rst:187]: The policy is described here:  https://mail.python.org/pipermail/python-committers/2017-March/004290.html

**Pull request merged**